services/job_runner.go

```diff
@@ -259,7 +261,7 @@ func executeRunAtBlock(
 		return jr, wrapExecuteRunAtBlockError(jr, errors.New("No unfinished tasks to run"))
 	}
 	offset := len(jr.TaskRuns) - len(unfinished)
-	prevResult, err := unfinished[0].Result.Merge(jr.Overrides)
+	prevResult, err := jr.Overrides.Merge(unfinished[0].Result)
 	if err != nil {
 		return jr, wrapExecuteRunAtBlockError(jr, err)
```

Comment:
It looks like this no longer applies the result with error from the old L305-306

Reply:
This is now done by `resumeJob` on the `job_runs_controller`.